docs/state.md

```diff
@@ -1,3 +1,56 @@
 # State
 
-todo
+- [state](#state) -  The `state` contains the necessary properties for rendering each part of a prompt in the terminal.
+
+| **term** | **description** |
+| --- | --- |
+| `type` | It shows the type of prompt created if specified in prompt |
+| `name` | It shows name of the prompt given by user|
+| `index` | It will show index of the prompt, by default `0` |
+| `message` | It shows the message to display in prompt created, if specified in prompt |
+| `header` | The header to be display in created prompt |
+| `footer` | The footer to be display in created prompt |
+| `error` | The error message to be shown if prompt is cancelled |
+| `hint` | It will contain the hint specified by the user in prompt, by default `''` |
+| `input` | It will contain the `stdin` if given, by default `''` |
+| `cursor` | The position of the console cursor to print |
+| `index` | The index value given to the prompt, by default `0` |
```

Comment:
`index` is listed twice

Reply:
Hi Brian, I have removed the duplicate index row from the doc